tests/configlet/util/generic_patch.py

```diff
@@ -0,0 +1,217 @@
+#! /usr/bin/env python
+
+import argparse
+import json
+import jsonpatch
+import fnmatch
+import os
+import re
+import sys
+import time
+import yaml
+
+from helpers import *
+from common import *
+
+if os.path.exists("/etc/sonic/sonic-environment"):
+    from mock_for_switch import config_reload, wait_until
+else:
+    from tests.common.config_reload import config_reload
+    from tests.common.utilities import wait_until
+
+CMD_APPLY_HACK = "config apply-patch -n -i '' -v {}"
```

Comment:
`-i ''` will ignore the whole config from validation. There is no need for `-n` option which just ignores the tables without YANG.

Reply:
It is just the HACK command. It will be gone entirely soon. So a redundancy is no-op